src/Truck.java

```diff
@@ -0,0 +1,17 @@
+public class Truck extends Vehicle {
+
+    public Truck(String modelName, int wheelsCount) {
+        super(modelName, wheelsCount);
+        checkEngine();
+        checkTrailer();
+    }
+    private void checkEngine() {
+        System.out.println("Проверяем двигатель " + modelName);
```

Comment:
нужно уточнять реализацию, чтобы был виден полиморфизм
```suggestion
        System.out.println("Проверяем двигатель грузовика " + modelName);
```